generate.js

```diff
@@ -12,11 +12,22 @@ ncp('static', 'out/static', err => {
   }
 })
 
+// slice(0) is to make new array, so it will not modified even though orgs is modified
+const orgsTemp = orgs.slice(0)
+const noLeader = orgsTemp.filter((org, index) => {
+  if (!org.leaders.length) {
+    orgs.splice(orgs.findIndex(o => o.id === orgsTemp[index].id), 1)
```

Comment:
I bet it is simpler to just run `filter` on orgs directly and use `noLeader.push` here.

Reply:
so, i need to change orgs from const to let?

Reply:
You can copy it.
Resulting 2 new array, withLeader & noLeader.

Reply:
We might have another use for the original orgs variable